sound/soc/intel/boards/bdw-rt5677.c

```diff
@@ -342,7 +349,11 @@ static struct snd_soc_dai_link bdw_rt5677_dais[] = {
 		.dpcm_playback = 1,
 		.dpcm_capture = 1,
 		.init = bdw_rt5677_init,
+#if IS_ENABLED(CONFIG_SND_SOC_SOF_BROADWELL)
+		SND_SOC_DAILINK_REG(ssp0_pin, be, dummy),
```

Comment:
why is this not the same as in broadwell.c 
SND_SOC_DAILINK_REG(ssp0_pin, codec, platform), 

??

Reply:
@plbossart this needs to change. @xiulipan has not updated the PR based on our findings last week

Reply:
@plbossart @ranj063 We find the issue why the dummy can no be found in ASoC. And we will fix in that way instead to link to SOF DAI